src/interfaces/IUniversalRewardsDistributor.sol

```diff
@@ -24,18 +24,18 @@ interface IUniversalRewardsDistributor {
     function pendingRoot() external view returns (uint256 submittedAt, bytes32 root, bytes32 ipfsHash);
     function claimed(address, address) external view returns (uint256);
 
+    function submitRoot(bytes32 newRoot, bytes32 ipfsHash) external;
```

Comment:
Putting functions in the order in which they appear in the contract, non-view functions being grouped by:
- callable by updaters
- callable by everyone
- callable by the owner